Makefile

```diff
@@ -8,7 +8,7 @@ zip: deps build
 
 new:
 	mkdir $(adapter)
-	cp -r example/ $(adapter)
+	cp -r example/* $(adapter)
```

Comment:
What is exactly wrong with this? It works on my machine (mac os). 

If I use `/*` then the cp command will not copy the `.eslinntrc.js` (hidden) file.

Reply:
On my machine (Ubuntu) it is copying the folder as well like: `[adapter name]/example/...` and not adding to `package.json`.

I found a way that should work for both of us, give it a try.